### DISCUSSION TITLE
Restore look and feel of pinned tabs of older versions, like Arc browser (or at least provide option)

### DISCUSSION THREAD

**Discussion by michaelpulis** [zen-browser/desktop#2955] [General]
Hey!

My current set up is one window, two workspaces. One is personal, and one is for work. In this setup, I have no use for essential tabs, and I was perfectly happy with pinned tabs as they allowed me to perfectly replicate my set up in arc browser. I do not wish to open more than one window, which is why I was happy to use multiple workspaces instead of multiple profiles.

After this latest update however, the look and feel of pinned tabs has been reverted to that of a normal browser, and only essential tabs get the look and feel of small buttons, I think. 

Will there be an update to revert this or to allow pinned tabs to have their old look-and-feel for people with my kind of setup?

Thanks,
Michael

**Comment by blakegearin** [comment]
Have you tried the "Add to Essentials" functionality?
